servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java

```diff
@@ -302,6 +302,15 @@ public synchronized void updateLastUpdatedTimeMs() {
   }
 
   /**
+   * Sets the last updated time of the worker in ms. Should only be called by tests.
```

Comment:
the relevant tests should probably use whitebox instead and this method should be removed